apps/aecore/src/aec_sync.erl

```diff
@@ -50,6 +52,15 @@ get_generation(PeerId, Hash) ->
 set_last_generation_in_sync() ->
     gen_server:cast(?MODULE, set_last_generation_in_sync).
 
+ask_all_for_node_info() ->
```

Comment:
Should it be exposed by HTTP?

Reply:
I wouldn't make this part of the node at all, it's a reporting functionality IMO.

Reply:
I'd rather keep it as an internal API which we call on demand. My concern would be an abuse of this API.